source/__tests__/integration/footer.test.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+describe('Sinopia Profile Editor Footer', () => {
+
+  describe.each`
+    pageUrl
+    ${'http://127.0.0.1:8000'}
+    ${'http://127.0.0.1:8000/#/profile/create/'}
+  `('header titles and links', ({pageUrl}) => {
```

Comment:
note this clever "tables" approach for repeating blocks of tests under diff conditions.  You can also do it for a single test.  https://jestjs.io/docs/en/api